src/components/services/ServiceBlock.jsx

```diff
@@ -1,16 +1,189 @@
-import React from 'react'
+import React, { useState } from 'react'
+import { Accordion } from 'react-bootstrap';
+import 'bootstrap/dist/css/bootstrap.min.css';
 import './ServiceBlock.css'
 
-import ServiceFilter from './serviceFilter/ServiceFilter';
-import Service from './service/Service';
+import ServiceItem from './serviceItem/SevriceItem';
+import pageText from "../text"
+
+
+const ServiceBlock = () => { 
+    const allServiceList = pageText().map((el) => {
+        return (<ServiceItem data={el}/>)
+    })
+    const splicedList = allServiceList.slice(0,6)
+
+    let [shownServices, changeService] = useState(splicedList)
+    let [moreClass, setMore] = useState("more")
+    let [chosenFilterClass, setFilterClass] = useState(["","",""])
+
+    function setFilter (e) {
+        setFilterClass(["","",""])
+        setMore("hidden")
+
+        if(e.target.id === "cosmetic")  setFilterClass(["chosen","",""]) 
```

Comment:
It's working, but a lot of hard coding stuff. Let's imaging that we need to have 10 categories and in current code we need to have 10 if, it's not good, is it?

To avoid this, let`s declare filter as separate variable
`    let filter = [
        {
            text: '1',
            id: 'cosmetic',
            activeColor: ''
        },
        {
            text: '2',
            id: 'laser',
            activeColor: ''
        },
        {
            text: '3',
            id: 'figure',
            activeColor: ''
        }];`
        
       then let's change remove ifs from setFilter and add 
       `filter = filter.map(f => f.id === e.target.id ? 'chosen' : '');`
       
       and now you can use it
       `<ul>
                            {filter.map(f => <li id={f.id} className={f.activeColor.length > 0 ? "chosen" : ""} onClick={setFilter}>{f.text}</li>)}
                        </ul>`